### ISSUE TITLE
Data storage location

### ISSUE THREAD

**Issue by Gitoffthelawn** [einaregilsson/Redirector#36]
I got a chance to test v3.0.4.  Looks great and much more powerful! :+1:

I'll file bug reports / etc. as separate Github issues as I find them.  If you would prefer a single report, just let me know.

For this issue: In Firefox, where should the Redirector data be stored?  Redirector is now storing data in 'browser-extension-data\redirector@einaregilsson`.  Is there a spec that defines this?

It seems like every extension stores its data in a different place nowadays.  What's the standard?

**Comment by einaregilsson** [comment]
This location, browser-extension-data/redirector@einaregilsson.com is where the new Firefox extension api, WebExtensions, ( https://wiki.mozilla.org/WebExtensions ) will store its data for the chrome.storage api. My plan is to move to using that api in the next version of Redirector, so I moved the data there now so it would be ready. When I move to WebExtensions I won't have direct file system access, so I needed to do this now if I wanted to automatically migrate people's redirects.
